tests/integrations/google_ai_studio/google_ai_studio_test.py

```diff
@@ -119,3 +119,34 @@ async def test_content_generation_async(client):
     # TODO: Re-enable after dictify is fixed
     # assert_correct_output_shape(call.output)
     # assert_correct_summary(call.summary, trace_name)
+
+
+@pytest.mark.retry(max_attempts=5)
+@pytest.mark.skip_clickhouse_client
+def test_send_message(client):
+    import google.generativeai as genai
+
+    genai.configure(api_key=os.getenv("GOOGLE_GENAI_KEY"))
+    model = genai.GenerativeModel(model_name="gemini-1.5-pro", tools="code_execution")
+    chat = model.start_chat()
+    chat.send_message(
```

Comment:
are you gonna support the async and streaming cases too?

Reply:
Added support for the async case, however, if `stream=True`, then a bug is being encountered where the run doesn't complete; here's a [sample trace](https://wandb.ai/geekyrakshit/test-google-ai-studio/weave/calls/0192d51c-987e-7950-b690-d36382f1bb97).

Reply:
did you look into why that's happening?